packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@emotion/css';
+import { navigate } from '@reach/router';
+import SpacingsStack from '@commercetools-uikit/spacings-stack';
+import SpacingsInline from '@commercetools-uikit/spacings-inline';
+import Link from './link';
+
+const flatStyle = css`
+  border: 1px solid #cccccc;
```

Comment:
All the colors, sizes, etc must be derived from the design system. Please don't forget about that.